src/main/java/org/jenkinsci/plugins/workflow/steps/MailStep.java

```diff
@@ -139,6 +145,14 @@ public StepExecution start(StepContext context) throws Exception {
         @Override
         protected Void run() throws Exception {
             MimeMessage mimeMessage = buildMimeMessage();
+            // JENKINS-53305 - contextClassLoader can _sometimes_ be null, which causes the obtuse error message
+            // javax.activation.UnsupportedDataTypeException: no object DCH for MIME type multipart/mixed;
+            // I can't see why this would ever be valid and don't know the cause, but if we encounter it, then set
+            // it to the uberclass loader
+            if (Thread.currentThread().getContextClassLoader() == null) {
+                LOGGER.log(Level.WARNING, "contextClassLoader is null - re-setting to uberClassLoader");
+                Thread.currentThread().setContextClassLoader(Jenkins.getInstance().getPluginManager().uberClassLoader);
```

Comment:
No, this is _not_ the normal `contextClassLoader` in Jenkins (surprisingly). Rather use `Jenkins.class.getClassLoader()`.